buildSrc/src/main/java/com/google/firebase/gradle/plugins/FiresiteTransformTask.kt

```diff
@@ -21,10 +25,15 @@ import org.gradle.api.tasks.TaskAction
  *  **Please note:**
  *  This task is idempotent- meaning it can safely be ran multiple times on the same set of files.
  */
+@CacheableTask
```

Comment:
Interesting, aren't tasks cacheable by default?

Reply:
Yeah, but having this explicit annotation forces you to label the annotations properly. For example, without this annotation, you could compile without the `PathSensitive` attribute (which helps solidify *how* the task should be looked at and cached).

Reply:
T.I.L. 👍